packages/devtools_app/lib/src/inspector/flutter/inspector_screen.dart

```diff
@@ -98,6 +98,8 @@ class _InspectorScreenBodyState extends State<InspectorScreenBody>
       controller: detailsTreeController,
       isSummaryTree: false,
     );
+    final screenSize = MediaQuery.of(context).size;
+    final aspectRatio = screenSize.width / screenSize.height;
```

Comment:
I suggest that we decide which layout to use just based on the width of the app, instead of the screen ratio.  Other parts of the app, like the narrow-width buttons and the tab bar use this heuristic already.

Take a look at how DevToolsScaffold determines the narrow width mode.

Reply:
What Albertus is suggesting matches what we did in the dart:html app.
The reason I think this case is different from the button case is this case isn't just about whether content fits. It is about what content looks "better". For a 1000x2000 window I would want us to split vertically because in that case having two 1000x1000 areas is "better" than two 500x2000 areas as both windows are easy to interact with. On the other hand for a 1000x500 window I would want us to split into two 500x500 windows not two 1000x250 windows.

Reply:
SGTM.